examples/jws_sign_detached_payload_example_test.go

```diff
@@ -17,7 +17,15 @@ func Example_jws_sign_detached_payload() {
 		return
 	}
 
-	serialized, err := jws.Sign(nil, jws.WithKey(jwa.HS256(), key), jws.WithDetachedPayload([]byte(payload)))
+	// If you plan to transmit your payload without base64-encoding (RFC 7797),
+	// it's best to set `b64: false` so libraries can act accordingly (e.g. the
+	// popular NodeJS `jose` library requires it set to false in order to verify
+	// a plaintext payload.
+	hdrs := jws.NewHeaders()
+	hdrs.Set("b64", false)
+	hdrs.Set("crit", "b64")
```

Comment:
Also, I'm fairly new to both Go and JWx stuff, but I think this `crit` header is ignored?